docs/guides/integration/dependency-bots.md

```diff
@@ -16,7 +29,7 @@ dependencies) by enabling the
 [`lockFileMaintenance`](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance)
 option:
 
-```json5 title="renovate.json5"
+```jsx title="renovate.json5"
```

Comment:
Why the change to jsx here?

Reply:
- `json5`:

![Screenshot From 2024-10-09 21-10-28](https://github.com/user-attachments/assets/e6f24ef0-dca3-4d9f-98b0-8b5cdf22ddd1)

- `jsx`:

![Screenshot From 2024-10-09 21-10-36](https://github.com/user-attachments/assets/57213a24-46b0-42bc-85a4-7d2e355f3e10)

Reply:
Interesting catch!